scipy/sparse/__init__.py

```diff
@@ -237,6 +237,9 @@
 from .extract import *
 from ._matrix_io import *
 
+# For backward compatibility with v0.19. This function is marked as deprecated.
```

Comment:
`csgraph` is a module, not a function. I'd just leave the bit about back-compat.